package.json

```diff
@@ -38,7 +38,7 @@
     "lint": "jscs . && jshint . --exclude-path .gitignore",
     "standalone": "browserify index.js --standalone Pjax > pjax.js",
     "build-debug": "browserify index.js --debug --standalone Pjax > pjax.js",
-    "tests": "tape -r ./tests/setup.js ./tests/**/*.js",
+    "tests": "tape -r ./tests/setup.js ./tests/lib/*.js ./tests/lib/proto/*.js",
```

Comment:
This change should not be necessary. The `**` glob should include everything under `./tests/`, including all subdirectories.

Reply:
I thought so to, but it doesn't seem to be working even on TravisCI:
* Before the change, only 40 tests ran: https://travis-ci.org/MoOx/pjax/jobs/332062989
* After the change, 77 tests ran: https://travis-ci.org/MoOx/pjax/jobs/332428661

[From what I've read](https://stackoverflow.com/a/1691202), `globstar` needs to be explicitly enabled so it may not be portable.

Reply:
if that can help, glob not wrapped in string can fail. I would recommend you try using `"tests": "tape -r ./tests/setup.js \"./tests/**/*.js\"",`

Reply:
(can fail depending on the bash used to run the command)

Reply:
Confirmed that wrapping the arg in quotes works on both my local environment (OS X) and [TravisCI](https://travis-ci.org/MoOx/pjax/jobs/332473782). Updated.